packages/shared/components/FormFieldWrapper.tsx

```diff
@@ -92,22 +72,33 @@ class FormFieldWrapper extends React.PureComponent<FormFieldWrapperProps, {}> {
         {errors.map((error, index) => (
           <li
             key={index}
-            id={this.getErrorId(error, id)}
+            id={getErrorId(error, id)}
             className={cx(liReset, padding("top", "xxs"))}
           >
             {error}
           </li>
         ))}
       </ul>
     );
-  }
+  };
 
-  private getDescribedBy(hintContent, errors) {
+  const getDescribedBy = (hintContent, errors) => {
     if (hintContent && errors) {
       return `${hintContent} ${errors}`;
     }
     return errors || hintContent;
-  }
-}
+  };
+
+  return (
+    <>
+      {children({
+        getValidationErrors: getValidationErrors(errors, id),
+        getHintContent: getHintContent(hintContent),
+        describedByIds: getDescribedBy(getHintContentId(), getErrorIds()),
+        isValid: !errors || (errors && errors.length === 0)
+      })}
+    </>
+  );
+};
```

Comment:
We've been using React.memo() for components that were previously a `PureComponent`. 

Could you wrap this export below with React.memo()?

`export default React.memo(FormFieldWrapper); ` 

You may need to update the snapshot tests afterward - `npm run test -- -u`